config/rbac/kustomization.yaml

```diff
@@ -9,7 +9,6 @@ resources:
 - role_binding.yaml
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
-- nonadminbackup_editor_role.yaml
```

Comment:
I think this is not necessary, it is a subset of `config/rbac/role.yaml`. Am I right to remove this?